### PR TITLE
fixing broken gracc link

### PR DESCRIPTION
**Pull request by lmichael107**
'Not sure if there's a different GRACC page we'd like to point to, as we don't have anything specific to CC* contributions or researchers benefitting, but the prior link to home was incorrect.

### CODE REVIEW

**Comment by osg-cat** [comment]
If we are just pointing to the GRACC home page, we should use `https://gracc.opensciencegrid.org/` — yes, there are redirects when you go there, such that the URL ends up longer, but the short URL is arguably the front door.

File: campus-cyberinfrastructure.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The Open Science Grid consortium operates the Open Science Pool (OSPool). Any re
...
+In 2020, these campuses contributed to the OSPool over [292 millon core hours to researchers using distributed high throughput computing (dHTC).](https://gracc.opensciencegrid.org/d/000000074/gracc-home?orgId=1) These core hours supported over 180 projects in fields of study ranging from  medical sciences to evolutionary sciences and from  bio statistics to physics. This campus support throughout the United States contributed to the advancement of science and to researchers both on and off their campuses. The campuses currently contributing CC*-funded resources are shown in the map below; every month, the OSG is working to help additional campuses to join this effort to support open science by sharing their resources:
```
Comment (lmichael107) [inline campus-cyberinfrastructure.md:15]:
```suggestion
In 2020, these campuses contributed to the OSPool over [292 millon core hours to researchers using distributed high throughput computing (dHTC).](https://gracc.opensciencegrid.org/) These core hours supported over 180 projects in fields of study ranging from  medical sciences to evolutionary sciences and from  bio statistics to physics. This campus support throughout the United States contributed to the advancement of science and to researchers both on and off their campuses. The campuses currently contributing CC*-funded resources are shown in the map below; every month, the OSG is working to help additional campuses to join this effort to support open science by sharing their resources:
```

Comment (lmichael107) [reply campus-cyberinfrastructure.md:15]: Addressing @osg-cat's comment on the 'right' URL for GRACC home page.

**Review by lmichael107 — COMMENTED** [review]
Fixed. See below.

**Pull request merged**